src/views/pages/modifierIdentifiantGestionnaireReseauPage/ModifierIdentifiantGestionnaireReseau.tsx

```diff
@@ -51,16 +78,40 @@ export const ModifierIdentifiantGestionnaireReseau = ({
           </div>
 
           <input type="hidden" name="projetId" value={projet.id} />
+          {listeGestionnairesRéseau && listeGestionnairesRéseau.length > 0 && (
+            <div>
+              <Label htmlFor="codeEICGestionnaireRéseau">Gestionnaire de réseau</Label>
+              <Select
+                id="codeEICGestionnaireRéseau"
+                name="codeEICGestionnaireRéseau"
+                onChange={(e) => handleGestionnaireSéléctionné(e)}
+                defaultValue={projet.gestionnaireRéseau?.codeEIC || ''}
+              >
+                {!projet.gestionnaireRéseau?.codeEIC && <option selected disabled hidden></option>}
+                {listeGestionnairesRéseau.map(({ codeEIC, raisonSociale }) => (
+                  <option value={codeEIC} key={codeEIC}>
+                    {raisonSociale} (code EIC : {codeEIC})
+                  </option>
+                ))}
+              </Select>
+            </div>
```

Comment:
je sais pas si la <div> apporte grand chose, ptet qu'on peut directement passer par une balise vide

```suggestion
            <>
              <Label htmlFor="codeEICGestionnaireRéseau">Gestionnaire de réseau</Label>
              <Select
                id="codeEICGestionnaireRéseau"
                name="codeEICGestionnaireRéseau"
                onChange={(e) => handleGestionnaireSéléctionné(e)}
                defaultValue={projet.gestionnaireRéseau?.codeEIC || ''}
              >
                {!projet.gestionnaireRéseau?.codeEIC && <option selected disabled hidden></option>}
                {listeGestionnairesRéseau.map(({ codeEIC, raisonSociale }) => (
                  <option value={codeEIC} key={codeEIC}>
                    {raisonSociale} (code EIC : {codeEIC})
                  </option>
                ))}
              </Select>
            </>
```

Reply:
Jusqu'à maintenant les label+input sont dans des div et ce pour les disposer les uns par rapport aux autres dans une flex box, donc je vais laisser comme ça pour garder une cohérence. Qu'est-ce que ça apporterait de retirer le div ?

Reply:
En vérité ça ne changerait rien, div n'a aucune valeur sémantique. Mais oui gardons une cohérence :)